modules/ROOT/pages/6.1-release-notes.adoc

```diff
@@ -0,0 +1,1121 @@
+
+
+=== Preview and Insert Template dialogs now display the correct content background color when using dark skins
+
+Previously, when {productname} was in any dark mode (ie `skin_url` pointed to a dark themed skin), choosing *View > Preview* presented a background in the preview that was not consistent with the dark themed content.
+
+Choosing *Insert > Insert template…* produced the same inconsistent presentation in the *Insert template…* dialog.
+
+This update corrects this inconsistency. As of {productname} 6.1, no matter the current mode, the *Preview* and *Insert template…* dialogs now use the same background color as the current {productname} editor window.
+
+
+=== Fixed incorrect word breaks in {productname} menus when the menu presented with a scrollbar 
+
+In some browsers, including Safari and Firefox on macOS or Linux, items in {productname} menus wrapped incorrectly when the browser window was small enough to cause the {productname} menu to present with a scroll-bar.
+
+In this circumstance, the menu item wrapped within a word, pushing individual characters on to an extra line.
+
+Despite the visual mis-representation, the menus still worked as expected.
+
+With this update, the visual mis-representation has been fixed. If {productname} menus must wrap, because of limited view space, the wrapping occurs at word boundaries, not within words.
+
+
+=== The InsertLineBreak command did not replace selected content
+
+The `InsertLineBreak` command, as its name suggests, adds a line-break at the insertion point.
+
+Previously, however, if there was material selected in a {productname} document and the `InsertLineBreak` command was issued, the added line-break did not replace the selected material. Instead, it, incorrectly, appended the line-break to the selection.
+
+This update corrects this. Running `InsertLineBreak` when a {productname} document has material selected now results in the added line-break replacing the selected material.
+
+
+=== Delete or Backspace did not work if the selection being deleted contained a `contenteditable="false"` element at the beginning or end of a document
+
+If the first or last element in a {productname} document had a `contenteditable="false"` property and material within that element was selected, pressing the Delete or Backspace key did not delete the material inside the element with the `contenteditable="false"` property.
+
+This update corrects this. In {productname} 6.1 material inside elements with a `contenteditable="false"` property show they have been selected and are deleted when the Delete or Backspace key is pressed, as expected.
+
+
+=== `uploadImages` no longer triggers two change events if there is a removal of images on upload
+
+Previously, if an image was uploaded and then immediately deleted, the removal triggered a second change event, which, in turn, changed the undo stack.
```

Comment:
This isn't quite correct... it was if an image failed to upload and was flagged for removal that the second change event occurred.

Reply:
Edited sentence in *`uploadImages` no longer triggers two change events* section:

-Previously, if an image was uploaded and then immediately deleted, the removal triggered a second change event, which, in turn, changed the undo stack.
+Previously, if an image failed to upload and was flagged for removal, the flagging triggered a second change event, which, in turn, changed the undo stack.